docs/GettingStartedDocs/Contributors/RunningStaticAnalysis.md

```diff
@@ -0,0 +1,42 @@
+# Getting started with running static analysis tools
+
+## scan-build
+
+*Note: scan-build only works with clang and gcc and as such is only currently supported for linux builds*
+
+1. Install scan-build-7 (or whatever version matches your version of clang)
+
+    ```
+    $ apt install clang-tools-7
+    ```
+
+2. (Optional) Create symlinks for scan-build and scan-view to avoid needing to specify a version. If you skip this step, just add the clang version to each `scan-*` command e.g. `scan-build-7 ...`
+
+    ```{bash}
+    $ ln -s /usr/bin/scan-build /usr/bin/scan-build-7
+    $ ln -s /usr/bin/scan-view /usr/bin/scan-view-7
+    ```
+
+3. Run scan-build for cmake and make build commands
+
+    For example:
+
+    ```{bash}
+    $ scan-build cmake .. -DHAS_QUOTE_PROVIDER=OFF
+    $ scan-build make -j`nproc`
+    ```
+
+*Note: scan-build also works fine with the cmake ninja generator*
```

Comment:
We generally prefer Ninja as it's required for Windows, and works much better on Linux than GNU Make. I'd recommend just updating the example to use Ninja.